oat-sa/extension-tao-itemqti-pci
# fix math entry interaction in preview mode

Ticket: https://oat-sa.atlassian.net/browse/MS-135

**Issue**

MathEntryInteraction does not show test taker the response in item previewer.

**What's in the PR?**

* State format in math entry interaction has been fixed, inspired by Likert PCI
* Redundant assertions about setting and getting response has been removed from the unit tests about setting and getting states. Those assertions are already covered at `L:231` and `L:287` plus the assertions were misleading as we were comparing the response to state

**How to test**

* Check out the branch 
* Create an item with math entry interaction and few more items (you will see why in step 4)
* Create a test, publish it and assign it to a TT (or enable guest mode)
* Start the delivery and add a response to math entry interaction
* Navigate back and forth from math entry interaction and when you come back to the item with math entry interaction, the state of the item should be restored (You should see your response inside the interaction)
* Submit the delivery
* Login as admin in tao and go to result tab
* Open the result of the delivery execution you just finished
* Review the item with math entry interaction
* MAth entry interaction should show your response

**Run unit tests**

* `cd tao/views/build`
* `npx grunt connect:test qunit:single --test=/qtiItemPci/views/js/test/mathEntryInteraction/test.html`
